### PR TITLE
fix(storage): MkdirAll 0644 => 0755

### PR DESCRIPTION
**Pull request by ccf5maj37**
Creating a folder with 0644 permissions will cause file creation to fail in the case of a non-root user.

### CODE REVIEW

File: pkg/storage/storagedb.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func New(options *Options) (*StorageDB, error) {
 		}
 		dbpath := filepath.Join(options.DbPath, xid.New().String())
 
-		if err := os.MkdirAll(dbpath, 0644); err != nil {
+		if err := os.MkdirAll(dbpath, 0755); err != nil {
```
Comment (dogancanbakir) [inline pkg/storage/storagedb.go:55]: Let's update this with `permissionutil.ConfigFolderPermission`. See: https://github.com/projectdiscovery/utils/blob/f110843046dd80c714ced0ba0857f0602e9f8c6f/permission/permission_file.go#L48

**Review by dogancanbakir — CHANGES_REQUESTED** [review]
@ccf5maj37 Thanks for the PR! I've left a comment for you.

File: pkg/storage/storagedb.go (hunk 1 above)
Comment (ccf5maj37) [reply pkg/storage/storagedb.go]: Thanks for the suggestion, I've changed it.

**Review by dogancanbakir — APPROVED** [review]
LGTM

**Pull request merged**